test/verify/check-storage-stratis

```diff
@@ -0,0 +1,370 @@
+                                    expect=lambda: (self.dialog_is_present('disks', dev_1) and
+                                                    self.dialog_is_present('disks', dev_2) and
+                                                    self.dialog_check({"name": "pool0"})))
+        self.dialog_set_val("disks", {dev_1: True, dev_2: True})
+        b.assert_pixels("#dialog", "create-pool")
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        b.wait_in_text("#devices", "pool0")
+        b.assert_pixels("#devices", "pool-row")
+
+        # Check that the next name is "pool1"
+        self.devices_dropdown("Create Stratis pool")
+        self.dialog_wait_open()
+        self.dialog_wait_val("name", "pool1")
+        self.dialog_cancel()
+        self.dialog_wait_close()
+
+        b.click('.sidepanel-row:contains("pool0")')
+        b.wait_visible('#storage-detail')
+
+        # Create two filesystems
+        b.click("button:contains(Create new Filesystem)")
+        self.dialog_wait_open()
+        self.dialog_set_val('name', 'fsys1')
+        self.dialog_set_val('mount_point', '/run/fsys1')
+        b.assert_pixels("#dialog", "create-fsys")
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        b.wait_in_text("#detail-content", "fsys1")
+        b.assert_pixels("#detail-content", "fsys-row")
+
+        b.click("button:contains(Create new Filesystem)")
+        self.dialog({'name': 'fsys2',
+                     'mount_point': '/run/fsys2'})
+        b.wait_in_text("#detail-content", "fsys2")
+
+        # Check that they have entries in fstab
+        self.assertNotEqual(m.execute("grep /run/fsys1 /etc/fstab"), "")
+        self.assertNotEqual(m.execute("grep /run/fsys2 /etc/fstab"), "")
+
+        # Rename one filesystem
+        self.content_dropdown_action(1, "Rename")
+        self.dialog({'name': "fsys1-renamed"})
+        b.wait_in_text("#detail-content", "fsys1-renamed")
+
+        # Destroy one filesystem
+        self.content_dropdown_action(1, "Delete")
+        self.dialog_wait_open()
+        b.assert_pixels("#dialog", "delete-fsys")
+        self.dialog_apply()
+        self.dialog_wait_close()
+        b.wait_not_in_text("#detail-content", "fsys1-renamed")
+
+        # Unmount and remount the other filesystem
+        self.content_dropdown_action(1, "Unmount")
+        self.content_tab_wait_in_info(1, 1, "Mount point", "The filesystem is not mounted")
+        self.content_row_action(1, "Mount")
+        self.dialog({})
+        self.wait_mounted(1, 1)
+
+        # Make a copy of the filesystem
+        self.content_dropdown_action(1, "Copy")
+        self.dialog_wait_open()
+        self.dialog_set_val('name', 'fsys2-copy')
+        self.dialog_set_val('mount_point', '/run/fsys2-copy')
+        b.assert_pixels("#dialog", "copy-fsys")
+        self.dialog_apply()
+        self.dialog_wait_close()
+        b.wait_in_text("#detail-content", "fsys2-copy")
```

Comment:
... and here, please check that it is mounted and that the file created above actually exists and has the expected content. I.e. the copy actually worked.

Reply:
Done